web/src/index.css

```
@@ -18,17 +18,45 @@
+  --semi-color-primary-light-hover: rgba(192, 96, 48, 0.15);
+  --semi-color-primary-light-active: rgba(192, 96, 48, 0.22);
 }
 
 body.sidebar-collapsed {
   --sidebar-current-width: var(--sidebar-width-collapsed);
 }
 
 body {
-  font-family: Lato, 'Helvetica Neue', Arial, Helvetica, 'Microsoft YaHei',
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Inter', Roboto,
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)